### PR TITLE
[Swift] Add fallback delete of chunks if bulk-delete is disabled

### PR DESCRIPTION
**Pull request by GlaciErrDev**

### CODE REVIEW

File: glance_store/_drivers/swift/store.py [hunk 1]
```diff
@@ -1014,10 +1025,14 @@ def _delete_chunks(self, connection, container, chunk_list):
             msg = _('No content received on account POST. '
                     'Is the bulk operations middleware enabled?')
             LOG.exception(msg)
-        LOG.debug(
-            "Done bulk delete of chunks with response: %s" %
-            parse_api_response(headers, body)
-        )
+            # Try to delete chunks in a sequential way as a fallback
+            self._delete_chunks_sequential(connection, container, chunk_list)
+            LOG.debug("Done bulk delete of chunks.")
```
Comment (galkindmitrii) [inline glance_store/_drivers/swift/store.py:1030]: wrong message

Comment (GlaciErrDev) [reply glance_store/_drivers/swift/store.py:1030]: why?

Comment (galkindmitrii) [reply glance_store/_drivers/swift/store.py:1030]: # Try to delete chunks in a sequential way as a fallback

Comment (GlaciErrDev) [reply glance_store/_drivers/swift/store.py:1030]: ahh ok, you're right!

**Pull request merged**